src/api/java/com/minecolonies/api/compatibility/CompatabilityManager.java

```diff
@@ -12,6 +12,7 @@
 import net.minecraft.block.BlockRedstoneOre;
```

Comment:
### Errors:
- Unused import import net.minecraft.block.BlockRedstoneOre;

Reply:
Are these here really unused? I don't trust it =D